docs/clip.md

```diff
@@ -0,0 +1,24 @@
+## Clip
+
+Depending on the data and domain, sometimes the series in the plot will extend into the axis. This can either be solved with a [Border](border.md), or the elements can be clipped.
```

Comment:
Maybe also point to some mdn docs about clipping people aren't familiar?

Reply:
Done